donkeycar/parts/tub_v2.py

```diff
@@ -78,8 +78,7 @@ def write_record(self, record=None):
         # Private properties
         contents['_timestamp_ms'] = int(round(time.time() * 1000))
         contents['_index'] = self.manifest.current_index
-        contents['_session_id'] = self.manifest.session_id
-
+        contents['_session_id'] = self.manifest.session_id[1]
```

Comment:
why the index of 1 here?  It's magic-number-ish.  Maybe worth a comment.

Reply:
Sorry, saw that comment too late... `session_id` now returns a tuple of the (number, session_id_string) which allows to simplify the code a lot, so we are picking the string here.